### DISCUSSION TITLE
Extreme M0 Values

### DISCUSSION THREAD

**Discussion by AlexMorello** [qMRLab/qMRLab#471] [General]
Dear @agahkarakuzu  and @mathieuboudreau,

My task is to account for mutual T1 and T2 effects. 
I've exported the T1 values for the slices in question, e.g this one:
![T1](https://user-images.githubusercontent.com/84672824/170875106-f17431e5-a046-446a-9732-5197cb65c2a5.png)

However, when switching to M0, the same output is as follows, with a deviation of 545:
![m00](https://user-images.githubusercontent.com/84672824/170876528-b1fe7631-65df-47e4-9268-d010bf5693cb.png)

**Comment by agahkarakuzu** [comment]
- Which model?
- Are T1 values accurate?
- What do you expect to see for M0? What do you mean by extreme?

**Reply by AlexMorello** [reply]
Wow, man, thanks for coming over so quickly. 

Here's the scheme I'm supposed to follow:

![model](https://user-images.githubusercontent.com/84672824/170877220-8dca78d6-7137-447f-b364-c78d02330eec.png)


The T1 seem to be sufficiently accurate. By extreme I mean the deviation, for one, - it's basically 'over 9000', or am I just getting the wrong idea?

**Reply by agahkarakuzu** [reply]
What I meant was which qmrlab model you used to fit your T1 data. Or did you implement something else in qmrlab and trying to make it work? I am not sure what you are doing with the level of explanation you provided. 

Regardless of these, you may consider M0 as a residual from the fit. There's not a standard scale, as you would get with T1. I suggest reading the docs for MTV module (https://qmrlab.readthedocs.io/en/master/mtv_batch.html) to have a sense of how M0 from these fits can be interpreted as proton density.

**Reply by AlexMorello** [reply]
> What I meant was which qmrlab model you used to fit your T1 data. Or did you implement something else in qmrlab and trying to make it work? I am not sure what you are doing with the level of explanation you provided.
> 
> Regardless of these, you may consider M0 as a residual from the fit. There's not a standard scale, as you would get with T1. I suggest reading the docs for MTV module (https://qmrlab.readthedocs.io/en/master/mtv_batch.html) to have a sense of how M0 from these fits can be interpreted as proton density.

Hey, thanks for getting back to me again, and for the link! Sorry about being vague :( I used this method initially: https://qmrlab.readthedocs.io/en/master/vfa_t1_batch.html
